.github/workflows/main.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
         # oldest and newest supported LLVM version
-        clang_version: [10.0.0, 14.0.0]
+        clang_version: [10.0.0, 15.0.7]
```

Comment:
Wouldn't we want to update the min version here?  I think the idea is to test with the oldest supported version and also with a current/newer version.

Reply:
even before this PR, we were not testing app link for 10.0.0.